.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "endOfLine": "auto",
+  "singleQuote": true,
+  "tabWidth": 4,
```

Comment:
In the ExtJS code we have a tabWidth of 3. Why did you choose 4 here?

Reply:
NI's eslint rules require a tabwidth of 4, see: https://github.com/ni/javascript-styleguide/blob/main/index.js#L66
Prettier doesn't recognize that automatically and it had to be defined here separately.

Reply:
Ok. That makes sense.